index.d.ts

```diff
@@ -30,6 +30,7 @@ declare module "electron-osx-sign" {
     install?: string;
     pkg?: string;
     scripts?: string;
+    'identity-validation'?: boolean;
```

Comment:
Ah thanks for spotting this! @ffflorian 🙌 
Would you mind moving this item to be in an alphabetical order?

Reply:
Sure :) I'll move the other one then, too